src/django/pfb_analysis/tasks.py

```diff
@@ -11,6 +11,7 @@
 
 from pfb_analysis.management.commands.import_results_shapefiles import add_results_geoms
 from pfb_analysis.management.commands.load_overall_scores import load_scores
+from pfb_analysis.management.commands.load_residential_speed_limit import load_speed_limit
```

Comment:
I think you may have forgotten to add `management/load_residential_speed_limit.py` to the commit

Reply:
This is currently failing for me with the following error:

```
Unknown command: 'load_residential_speed_limit'
Type 'manage.py help' for usage.
Updating job status: ERROR Failed See job logs for more details.
```

Reply:
Alas, so I did.  Just added it.